cms/djangoapps/contentstore/tasks.py

```diff
@@ -1204,31 +1204,43 @@ async def _validate_urls_access_in_batches(url_list, course_key, batch_size=100)
 
     for i in range(0, url_count, batch_size):
         batch = url_list[i:i + batch_size]
-        async with aiohttp.ClientSession() as session:
-            tasks = [_validate_url_access(session, url_data, course_key) for url_data in batch]
-            batch_results = await asyncio.gather(*tasks)
-            responses.extend(batch_results)
-            LOGGER.debug(f'[Link Check] request batch {i // batch_size + 1} of {url_count // batch_size + 1}')
+        batch_results = await _validate_batch(batch, course_key)
+        responses.extend(batch_results)
+        LOGGER.debug(f'[Link Check] request batch {i // batch_size + 1} of {url_count // batch_size + 1}')
```

Comment:
Should we leave this logging statement here?
Or should we update this logging statement to be recorded on datadog in stage and prod?

Reply:
What's involved in having it be recorded on Datadog?